src/views/Layout.jsx

```diff
@@ -15,33 +15,41 @@ import { useAuth, SignInButton, SignOutButton } from '../api/useAuth.jsx';
 export function Layout() {
 	const { user } = useAuth();
 	return (
-		<>
-			<div className="Layout">
-				<header className="Layout-header">
-					<h1>Smart shopping list</h1>
-					{!!user ? (
+		<div className="Layout">
+			{!!user ? (
```

Comment:
I'm curious what the double !! is for?

Reply:
I think !! evaluates something as a boolean, so in this case !!user would equal to true or false, rather than evaluating it as a truthy or falsy. In this case maybe just `user ?` would have been enough? But it was the way it was originally, so I just kept it. @aberger3647